### PR TITLE
106 feature temporally filter projects

### PR DESCRIPTION
**Pull request by alexjcalvillo**
I am confused about the locations markers in the initial map container have no "year" property and changing dates currently removes those markers. Something to address?

### CODE REVIEW

**Comment by krisstee** [comment]
Hey Alex! Let's take a look at tomorrow's meeting

File: frontend/src/components/panels/project_list_panel/ProjectListPanel.js [hunk 1]
```diff
@@ -23,8 +25,11 @@ class ProjectListPanel extends React.Component {
         meep_service.getProjectDetailsById(project_id)
             .then(selectProject)
             .then(this.props.dispatch)
-    }
+    };
+
     render() {
+        console.log(this.props.filters)
```
Comment (krisstee) [inline frontend/src/components/panels/project_list_panel/ProjectListPanel.js:31]: Looks like there's some console.log statements still here, haha. Can we also clean this up part?

Comment (alexjcalvillo) [reply frontend/src/components/panels/project_list_panel/ProjectListPanel.js]: I forgot to push that commit...somehow. Should be fixed now. Thanks.

**Review by chrisschaaf — APPROVED** [review]
Seems like it works for date filtering based on the random data generation, panel only is returning projects within the selected date range.

**Comment by chrisschaaf** [comment]
I had to fix up a merge conflict involving something I pushed that I'm pretty sure broke this feature. I replaced it with (f667206) the version of ProjectListPanel that you still had in this PR, and it is filtering properly with that.

**Pull request merged**